config/routes.rb

```diff
@@ -4,6 +4,8 @@ def put(*_args); end
 
   routing_helper = Insights::API::Common::Routing.new(self)
 
+  get '/health', :to => "status#health"
```

Comment:
This and the `status_controller.rb` file that inherits from the common controller would be the only things required to implement the healthcheck for each service.